.gitignore

```diff
@@ -15,4 +15,7 @@ $RECYCLE.BIN/
 *.msp
 
 # Local config AChecker
-include/config.inc.php
\ No newline at end of file
+include/config.inc.php
+
+# Vendor Folder
```

Comment:
Could you remove this 2 lines if they are specific for your local dev environment.

Reply:
I see. `vendor/` is created by PHP composer that is introduced in this branch. Please ignore my previous comment. Thanks.